tv_apps/api-demos-tv/manifest.webapp

```diff
@@ -0,0 +1,23 @@
+{
+  "name": "API Demos",
+  "description": "Demo app to show all APIs in TV",
+  "launch_path": "/index.html",
+  "type": "certified",
+  "developer": {
+    "name": "The Gaia Team",
+    "url": "https://github.com/mozilla-b2g/gaia"
+  },
+  "permissions": {
+    "desktop-notification":{}
+  },
+  "locales": {
+    "en-US": {
+      "name": "API Demos",
+      "description": "Demo app to show all APIs in TV"
+    }
+  },
+  "default_locale": "en-US",
+  "icons": {
+    "354": "/style/icons/developer.png"
+  }
+}
```

Comment:
Yes, we miss  "messages": [ { "notification": "/index.html" } ] in the first case. I will add it back.

For the [2], the connect is not missing. We do not support the app is killed after the notification shown. Once we changed the notification API to support multiple buttons. The IAC parts will be removed. And all of this parts go back to desktop notification case.